test/server/cards/03_TWI/units/KrakenConfederateTactician.spec.ts

```diff
@@ -0,0 +1,68 @@
+describe('Kraken Confederate Tactician', function () {
+    integration(function (contextRef) {
+        describe('Kraken Confederate Tactician\'s ability', function () {
+            it('should create two friendly Battle Droid tokens', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['kraken#confederate-tactician'],
+                    }
+                });
+
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.kraken);
+
+                const battleDroids = context.player1.findCardsByName('battle-droid');
+                expect(battleDroids.length).toBe(2);
+                expect(battleDroids).toAllBeInZone('groundArena');
+                expect(battleDroids.every((battleDroid) => battleDroid.exhausted)).toBeTrue();
+                expect(context.player2.getArenaCards().length).toBe(0);
+            });
+
+            it('should give +1/+1 to friendly tokens unit on attack', function () {
```

Comment:
We can merge this test into the above pretty easily by just starting Kraken in hand, playing him, then doing the attack tests. That also simplifies things a little since he'll create the tokens needed for the test

Reply:
How can I move easily to the next action phase in order to make kraken attacks? 
Or maybe you had in mind to use an ambush event card?

It's not yet super clear to me the motivation behind merging or splitting tests. I understand that it's reduce the github actions but sometimes splitting seemed ok in my previous PR 😅.

Reply:
I found it `context.moveToNextActionPhase();`

Reply:
My general rule of thumb is that if you can get multiple test cases together with minimal manual changes to game state like un-exhausting a unit that is attacking, it's good. But when you have to start manually moving cards around a lot or doing major changes to game state using test commands, it can potentially break game state and cause the tests to become invalid.